tensorflow/python/framework/fast_tensor_util.pyx

```diff
@@ -7,6 +7,18 @@ cimport numpy as np
 from tensorflow.python.util import compat
 
 
+def AppendFloat16ArrayToTensorProto(
+    # For numpy, npy_half is a typedef for npy_uint16,
+    # see: https://github.com/numpy/numpy/blob/master/doc/source/reference/c-api.coremath.rst#half-precision-functions
+    # Because np.float16_t dosen't exist in cython, we use uint16_t here.
+    # TODO: Use np.float16_t when cython supports it.
+    tensor_proto, np.ndarray[np.uint16_t, ndim=1] nparray):
+  cdef long i, n
```

Comment:
Can you run pyformat on this?

Reply:
Do you mean to use `def i, n` instead? I think `cdef` should be more efficient. 
By the way, in the file, all `AppendXXXXArrayToTensorProto` methods use `cdef` with `append` route, which might be designed on purpose.